visualization/app/codeCharta/ui/downloadButton/downloadButton.component.html

```diff
@@ -1,3 +1,3 @@
-<md-button class="toolbar-button" ng-click="$ctrl.downloadFile()">
+<md-button class="toolbar-button" ng-click="$ctrl.downloadFile()" title="Download cc.json">
```

Comment:
To me it's unclear what the benefit of the download button is. As a user, I already imported the cc.json. Why should I download that again? We should somehow clarify that it's only about potential modifications.

We might also only allow to download the cc.json if it's not identical to the original cc.json anymore.

We should not invest to much work into this though, since it's planned to rework the whole download logic.

Reply:
That is true. I just can't think of a short tooltip describing the use of this download button.

Let's just keep this for now and get rid of the download button in the new feature soon